### PR TITLE
style(auth-pages): general style and ui improvements

### PR DESCRIPTION
**Pull request by pakyardim**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import ReactDOM from 'react-dom/client';
 // import { Provider } from 'react-redux';
 import './assets/index.scss';
```
Comment (kaganecee) [inline src/index.js:6]: Please import local imports after global imports.

File: src/pages/Authentication/SignIn.js [hunk 2]
```diff
@@ -26,15 +56,20 @@ function SignIn() {
...
                 type="checkbox"
+                name="remember"
+                checked={formValues.remember}
+                onChange={handleChange}
                 className="checkbox-input absolute opacity-0 pointer-events-none"
               />
               <span className="checkbox-icon inline-block w-5 h-5 mr-2.5 rounded border border-gray-500 rounded"></span>
               Remember me
             </label>
             <a href="#">Forgot your password?</a>
```
Comment (kaganecee) [inline src/pages/Authentication/SignIn.js:67]: Can we use Link component instead of a element.

File: src/components/UI/PasswordInput.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React, { useState } from 'react';
 import { FaEye, FaEyeSlash } from 'react-icons/fa';
 
-function PasswordInput({className}) {
+function PasswordInput({ value, onChange, className }) {
```
Comment (kaganecee) [inline src/components/UI/PasswordInput.js:4]: Our function style must be const Roadmap = () => {... Can you refactor all of them. I will add eslint rules for it.

File: package.json [hunk 4]
```diff
@@ -7,6 +7,8 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.4.0",
+    "dotenv": "^16.0.3",
```
Comment (kaganecee) [inline package.json:11]: Do we use dotenv lib? I didn't see any import for it.

**Review by kaganecee — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (pakyardim) [reply package.json:11]: We don't.

**Pull request merged**